master/internal/api_user_intg_test.go

```diff
@@ -48,7 +48,9 @@ var (
 // called twice for the same resource, etc).
 func MockRM() *mocks.ResourceManager {
 	var mockRM mocks.ResourceManager
-	mockRM.On("DeleteJob", mock.Anything).Return(sproto.EmptyDeleteJobResponse(), nil)
+	mockRM.On("DeleteJob", mock.Anything).Return(func(sproto.DeleteJob) sproto.DeleteJobResponse {
```

Comment:
question: why need a function `func(sproto.DeleteJob)` instead of returning `sproto.EmptyDeleteJobResponse()` directly?

Reply:
because the response gets reused for everything that calls it, and so we don't get a new channel and 2/3 of the jobs block indefinitely.